cpp/test.yaml

```
@@ -0,0 +1,9 @@
+test:
+
+  tests:
+    - name: test-echo-cpp
+      harness: test-echo.py
```

Why test-echo? You are not sending anything, I'd leave the same structure as in psh tests:
```suggestion
    - name: hello-cpp
      harness: test-hello-cpp.py
```

---

My mistake, I made changes other than I intended.